layouts/partials/footer.html

```diff
@@ -62,7 +62,7 @@ <h3 class="widget-title">Form widget</h3>
 
 			<div class="col-md-6 widget">
 				<div class="widget-body">
-					<p>{{ .Site.Params.address }}</p>
+					<p>{{ if .Site.Params.address}}{{ .Site.Params.address }}{{ end }}</p>
```

Comment:
Not necessary at the moment, but consistent in case style changes in the future.